tests/conftest.py

```diff
@@ -140,6 +142,50 @@ def mock_dest(httpx_mock):
     )
 
 
+@pytest.fixture
+def mock_bulk_get_src(httpx_mock):
+    get_by_geid_url = ConfigClass.META_SERVICE + 'items/batch/'
+    mock_folder = FOLDER_DATA.copy()
+    mock_folder['id'] = str(SRC_FOLDER_ID)
+    mock_folder['name'] = 'src_folder'
+    mock_data = {'result': [mock_folder]}
+    httpx_mock.add_response(
+        method='GET', url=get_by_geid_url + f'?ids={str(SRC_FOLDER_ID)}', json=mock_data, status_code=200
+    )
+
+
+@pytest.fixture
+def mock_bulk_get_dest(httpx_mock):
+    get_by_geid_url = ConfigClass.META_SERVICE + 'items/batch/'
+    mock_folder = FOLDER_DATA.copy()
+    mock_folder['id'] = str(DEST_FOLDER_ID)
+    mock_folder['name'] = 'dest_folder'
+    mock_data = {'result': [mock_folder]}
+    httpx_mock.add_response(
+        method='GET', url=get_by_geid_url + f'?ids={str(DEST_FOLDER_ID)}', json=mock_data, status_code=200
```

Comment:
I would suggest adjusting to f-strings:
```suggestion
        method='GET', url=f'{get_by_geid_url}?ids={str(DEST_FOLDER_ID)}', json=mock_data, status_code=200
```